cmd/util.go

```diff
@@ -311,3 +313,21 @@ func strictDecodeFile(path, component string, cfg interface{}) error {
 
 	return errors.Trace(err)
 }
+
+// logHTTPProxies logs HTTP proxy relative environment variables.
+func logHTTPProxies() {
```

Comment:
Please add a unit test for this function

Reply:
Done.